packages/community-cli-plugin/src/commands/bundle/index.js

```diff
@@ -114,6 +114,17 @@ const bundleCommand: Command = {
       description: 'Path to the CLI configuration file',
       parse: (val: string): string => path.resolve(val),
     },
+    {
+      name: '--custom-resolver-options <string>',
+      description: 'Custom resolver options, format: key=value,key2=value2.',
+      parse: (val: string): Record<string, string> => {
+        return val.split(',').reduce((options, option) => {
+          const [key, value] = option.split('=');
+          options[key] = value;
+          return options;
+        }, {});
```

Comment:
Flow needs the type of the initial value in this `reduce` call, i.e. a typecast `({}: Record<string, string>)`.

Can I suggest this alternative? (**Please test in practice**)

```suggestion
        return Object.fromEntries(
          val.split(',').map(option => {
            const [key, value] = option.split('=');
            return [key, value];
          }),
        );
```